dot/network/service.go

```diff
@@ -253,6 +256,7 @@ func (s *Service) Start() error {
 	}
 
 	s.host.registerStreamHandler(s.host.protocolID+SyncID, s.handleSyncStream)
+	s.host.registerStreamHandler(s.host.protocolID+WarpSyncID, s.handleWarpSyncStream)
```

Comment:
What will happen if we receive a request now since we don't have implemented the warpSyncProvider yet? it will fail? Perhaps it's better to remove / comment this line until have the warpSyncProvider to correctly respond to this requests. 🤔

Reply:
you can return a empty warp proof or just close the incoming stream, should not be a problem for now, but yeah, just commenting the line should not be a problem.